client/lib/plans/features-list.js

```diff
@@ -84,6 +86,7 @@ export const FEATURES_LIST = {
 					),
 				},
 			} ),
+		getTrackingLabel: () => 'All Premium features',
```

Comment:
We're intentionally not translating those, right?

Reply:
Yes, for the reason explained in [my other comment](https://github.com/Automattic/wp-calypso/pull/37442/files#r345117830):

> we [...] would run into an issue of having multiple events recorded in Tracks that would virtually be the same but would have different labels depending on [the user's] locale.

This will eventually be removed since I'm planning to get rid of textual labels in favour of more versatile slugs.

Reply:
Cool, thanks!